app/javascript/components/tabs.jsx

```diff
@@ -107,7 +127,7 @@ export class Tab extends React.Component {
     }
     return (
       // eslint-disable-next-line
-      <div>
+      <div onKeyDown={this.spaceBarPress} tabIndex="0">
```

Comment:
So `spaceBarPress` is a tough name for that function, since you reject a whole bunch of calls if the button pressed wasn't the spacebar.  I think this would be much more clear if you inlined this code inside of `onKeyDown`, since then its less misleading.

`onKeyDown={({keyCode}) => if (keyCode === 32) this.togglePause()}`

Reply:
@thedelchop what if we had `if (keyCode === spacebar)`
I don't think it's inherently obvious keyCode 32 is the spacebar, so maybe that would make it clearer?